pyrootsOfTheCaribbean/plot_correlation_matrix.py

```diff
@@ -0,0 +1,165 @@
+import pandas
```

Comment:
this file is based on a very old structure of the repository and should be updated to be used with an option parser and the variable sets in `variable_sets/` instead of the `variable_info.py` file